external/implementations/samlconf-openam-impl/README.md

```diff
@@ -0,0 +1,69 @@
+| Does not use errorless Base64 encryption | Bindings 3.5.4 | A SAML protocol message is form-encoded by applying the base-64 encoding rules to the XML representation of the message and placing the result in a hidden form control within a form as defined by [HTML401] Section 17.
+| Does not validate that Relay States are less than or equal to 80 bytes| Bindings 3.4.3 | RelayState data MAY be included with a SAML protocol message transmitted with this binding. The value MUST NOT exceed 80 bytes.
+| Does not validate the \<Subject\> element on the \<AuthnRequest\> | Profiles 4.1.4.1 | Note that the service provider MAY include a <Subject> element in the request that names the actual identity about which it wishes to receive an assertion. This element MUST NOT contain any <SubjectConfirmation> elements. If the identity provider does not recognize the principal as that identity, then it MUST respond with a <Response> message containing an error status and no assertions.
+| Does not sign POST Logout Responses | Profiles 4.4.4.2 | The responder MUST authenticate itself to the requester and ensure message integrity, either by signing the message or using a binding-specific mechanism.
+| Does not use the same NameID given from the AuthRequest in the Logout Request | Profiles 4.4.4.1 | The principal MUST be identified in the request using an identifier that strongly matches the identifier in the authentication assertion the requester issued or received regarding the session being terminated, per the matching rules defined in Section 3.3.4 of [SAMLCore].
+| Does not respond with the SAML Request's relay state | Bindings 3.5.4 | If a "RelayState" value is to accompany the SAML protocol message, it MUST be placed in an additional hidden form control named RelayState within the same form with the SAML message.
+| Does not return a SAML Error Response when given a Non-Matching destination | Core 3.2.1 | If [Destination] is present, the actual recipient MUST check that the URI reference identifies the location at which the message was received. If it does not, the request MUST be discarded.
+
+## Steps to Test OpenAM's IdP
+#### Installing OpenAM
+* Download [Apache Tomcat](https://tomcat.apache.org/download-70.cgi).
+* Download the OpenAM .war files from [OpenAM Releases](https://github.com/OpenIdentityPlatform/OpenAM/releases).
+* Rename OpenAM-\<version\>.war to OpenAM.war.
+* Unzip Apache Tomcat and move all OpenAm .war files into \<tomcat\>/webapps.
+* Run the following commands to start tomcat:
+
+##### NIX
+```bash
+chmod -R 755 <tomcat>/bin
+export CATALINA_HOME="<tomcat>"
+sh <tomcat>/bin/startup.sh
+```
+##### Windows
+<!--- TODO -->
+```
+<In Progress...>
```

Comment:
❓ Will you be able to fill this out before this gets merged or what is the plan?

Reply:
I can get to that later today, I've been putting it off since I don't normally use windows for development